directory/src/App.js

```diff
@@ -67,12 +69,13 @@ const SearchBar = ({ onSubmit }) => {
   );
 };
 
-const renderIcon = (family, name) => (
-  <div className="Result-Icon-Container" key={name}>
-    <Icon family={family} name={name} className="Result-Icon" />
-    <h4 className="Result-Icon-Name">{name}</h4>
-  </div>
-);
+const renderIcon = (family, name) =>
+  family.includes('_meta') ? null : (
```

Comment:
Can we change the script that generates this data instead?

Reply:
Pretty sure, fixing the script will be the best choice

Reply:
AFAIK the meta files are needed to point to the correct font file, just like it's done for FA5: https://github.com/oblador/react-native-vector-icons/blob/master/directory/bin/generate-glyphmap-index.js So simply just copy paste FA5 and apply to FA6 would fix this issue

Reply:
ready for review

I returned the render method to original, and fixed the script that generates the glyphmaps, adding an exception if the file is a _meta, that was the source of the problem.

let me know if I need to change or improve something